core/domain/story_domain.py

```diff
@@ -463,6 +463,22 @@ def get_node_index(self, node_id):
                 return ind
         return None
 
+    def find_node_index(self, node_id):
```

Comment:
Why is this added here? I don't see it used in this PR -- is it a merge error?

Reply:
Yes, it was a merge error, removed it.